Source/WebCore/page/scrolling/ScrollingTreeFrameHostingNode.cpp

```diff
@@ -83,6 +83,13 @@ void ScrollingTreeFrameHostingNode::willBeDestroyed()
     removeHostedChildren();
 }
 
+void ScrollingTreeFrameHostingNode::removeHostedChild(RefPtr<ScrollingTreeNode> node)
+{
+    m_hostedChildren.remove(node);
+    if (node)
```

Comment:
Why not the if check for the line above?

Reply:
Fixed.